validator/json/schema.json

```diff
@@ -266,10 +266,12 @@
           "type": "string"
         },
         "do_not_record": {
-          "type":  ["boolean", "null"]
+          "type":  ["boolean", "string", "null"],
+          "description": "If this value is set to `true`, the speaker does not want a recording of any kind. We will remove all video staff and maybe even switch off all cameras"
         },
         "do_not_stream": {
-          "type":  ["string", "null"]
+          "type":  ["string", "boolean", "null"],
+          "description": "If this value is set to boolean `false`, the speaker is okay with a live stream of the event; and is aware that somebody in the internet might create third party recording"
```

Comment:
Should `with a live stream` be `with a recording but not with a live stream`?

Reply:
for the `schedule.json`, I would propose following assignment for a talk without recording + live-stream:

```json
"do_not_record": true,
"do_not_stream": true,
```

The following assignment is a talk without a recording, but with a livestream:
```json
"do_not_record": false,
"do_not_stream": true,
```